python/lsst/ts/MTAOS/config_schema.py

```diff
@@ -293,11 +293,13 @@
         properties:
             isr:
               type: object
+              additionalProperties: false
```

Comment:
Please help to explain this in document (or comment). This field name is too general to know its meaning (or how to use it). Thanks!

Or just rename this field to something more specific.

Similar thing for the `additionalProperties` in this `config_schema.py`.